pandas/core/frame.py

```diff
@@ -4624,8 +4694,15 @@ def query(self, expr: str, *, inplace: bool = False, **kwargs) -> DataFrame | No
         if not isinstance(expr, str):
             msg = f"expr must be a string to be evaluated, {type(expr)} given"
             raise ValueError(msg)
-        kwargs["level"] = kwargs.pop("level", 0) + 1
-        kwargs["target"] = None
+        kwargs: Any = {
```

Comment:
It appears to me there is no reason to construct a dict here anymore. Assuming that's correct, can you instead pass these as normal function arugments.